src/core/Slider/Slider.tsx

```diff
@@ -12,6 +12,47 @@ import { getBoundedValue } from '../utils/common';
 import { Track } from './Track';
 import { Thumb } from './Thumb';
 
+// based on https://usehooks.com/useEventListener/
+function usePointerEventListener(
+  eventName: string,
+  handler: (event: PointerEvent) => void,
+  element: HTMLElement | Document | undefined,
+) {
+  // Create a ref that stores handler
+  const savedHandler = React.useRef<(event: PointerEvent) => void>();
+
+  // Update ref.current value if handler changes.
+  // This allows our effect below to always get latest handler ...
+  // ... without us needing to pass it in effect deps array ...
+  // ... and potentially cause effect to re-run every render.
+  React.useEffect(() => {
+    savedHandler.current = handler;
+  }, [handler]);
+
+  React.useEffect(
+    () => {
+      // Make sure element supports addEventListener
+      const isSupported = element && element.addEventListener;
+      if (!isSupported) {
+        return;
+      }
```

Comment:
Is this needed? https://caniuse.com/addeventlistener

Could just do:
```suggestion
      if (!element) {
        return;
      }
```

Reply:
updated